README.md

```diff
@@ -61,43 +61,39 @@ from a command prompt in your project folder:
 
 **Note:** If when starting the application with Docker you get the error `UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte` this may be because you created the `.env`-file using PowerShell. This can be solved by deleting that file and creating a new one using a bash shell (for example Git Bash).
 
-### 3. Install and set up the environment
+### 3. Install and set up the environment using docker
 
-We recommend using Docker to install and run the application. However alternatively if you prefer, [you can install directly to your system environment instead](https://github.com/codebar/planner/blob/master/native-installation-instructions.md).
+We recommend using Docker to install and run the application. However alternatively if you prefer, [you can install directly to your system environment instead](./native-installation-instructions.md).
 
-#### Install using Docker
+Before you start, you will need to have Docker installed and running. You can [download it from the Docker website](https://docker.com/). Once downloaded, install and start the Docker application.
 
-Before you start, you will need to have Docker installed and running. You can [download it from the Docker website](https://docker.com/).
+Run `bin/dup` to build and create the docker container. This will also set up the Rails application within the container and start the container. You will only have to run this command once. After initial setup, use `bin/dstart` to start an existing container - using `bin/dup` will recreate an existing container and reset the database.
 
-The current Dockerfile and docker-compose were closely copied from [the Docker guide](https://docs.docker.com/compose/rails/).
+### 4. Start the app
 
-1. Run `bin/dbuild` to build and setup the docker environment.
-2. Run `bin/drake` to run all the tests and make sure everything works.
+Run `bin/dserver` to start the Rails server.
 
-**Note:** If you are using Windows, you can run `bin/dbuild` using Git Bash.
+*If you have previously stopped the container, you will have to first start the container using `bin/dstart`.
```

Comment:
`bin/dstart` didn't seem to work for me to get the container up and running again after closing it down. I was able to do `bin/dserver` to get it working again

Reply:
@KimberleyCook do you remember the exact steps you took? You ran `bin/dstop` and then `bin/start`?

Reply:
Might be worth noting that "running the container" is not the same thing as "running the rails http server in the container".

`dstart` will start the container - but the rails server won't be running
`dserver` will start the rails https server once a container is running

Reply:
@KimberleyCook I'm thinking this might have been your issue. I'm going to merge the PR for now, since you got it working eventually. But yes, running a container and running the rails server are two different things.